src/editorPreview/browserPreview.ts

```diff
@@ -258,7 +259,7 @@ export class BrowserPreview extends Disposable {
 				.then((selection: vscode.MessageItem | undefined) => {
 					if (selection) {
 						if (selection === OPEN_EXTERNALLY) {
-							vscode.env.openExternal(uri);
+							ExternalBrowserUtils.openInBrowser(givenURL, SettingUtil.GetConfig().customExternalBrowser);

```

Comment:
Wouldn't VS Code benefit from having this util and other components being able to also support this? Instead of this being in the live-preview repo?

Reply:
It seems that there was a feature request https://github.com/microsoft/vscode/issues/143465 that didn't get enough support from the community that dealt with this. I would imagine that there also are a few difficulties with shipping with a new npm package. For now, I think it's sufficient to just use `open` directly, since I'm just opening URLs and nothing too complicated (ie: other schemes).